### DISCUSSION TITLE
blog/proxy/sing-box-on-windows

### DISCUSSION THREAD

**Discussion by giscus** [hominsu/blog#12] [Announcements]
# blog/proxy/sing-box-on-windows

The blog discusses how to install and configure sing-box, a next-generation universal proxy platform, on Windows using direct download or package managers, and outlines setting it up as a Windows service for automatic execution using WinSW.

https://homing.so/blog/proxy/sing-box-on-windows

**Comment by alliwang** [comment]
Thanks,确实非常优雅~

**Comment by ArsNovaHHS** [comment]
问什么我把订阅链接粘过去结果显示The configuration file cound not be loaded. '=' is an unexpected token. The expected token is ';'. Line 5, position 93.呢？

**Reply by hominsu** [reply]
配置文件有问题

**Comment by ghost** [comment]
太雅了

**Comment by akiirui** [comment]
在Windows上完全不需要第三方软件来启动服务，只需要 Task Scheduler 即可。

要点：
- Triggers 设置为 `At startup`
- General - Secuity options 设置为 `Run whether user is logged on or not`, 勾选 `Do not store password`
